dbms/tests/queries/0_stateless/00502_custom_partitioning_local.sql

```diff
@@ -28,7 +25,7 @@ DROP TABLE test.not_partitioned;
 SELECT '*** Partitioned by week ***';
 
 DROP TABLE IF EXISTS test.partitioned_by_week;
-CREATE TABLE test.partitioned_by_week(d Date, x UInt8) ENGINE = MergeTree(toMonday(d), x, 8192);
+CREATE TABLE test.partitioned_by_week(d Date, x UInt8) ENGINE = MergeTree PARTITION BY toMonday(d) ORDER BY x;
```

Comment:
Is the index granularity from settings preserved after server restart (with the ATTACH query)?

Reply:
Yes, it will be preserved in the ATTACH query. In fact it will be preserved even if it wasn’t specified explicitly (because the default can change after table creation).